src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/BarnehagelisteConsumer.kt

```diff
@@ -1,52 +1,42 @@
 package no.nav.familie.ks.sak.barnehagelister
 
-import no.nav.altinnkanal.avro.ReceivedMessage
-import no.nav.familie.ks.sak.barnehagelister.domene.BarnehagelisteMottatt
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.ks.sak.barnehagelister.domene.Barnehagebarn
 import no.nav.familie.ks.sak.config.KafkaConfig
-import org.apache.kafka.clients.consumer.ConsumerRecord
 import org.slf4j.LoggerFactory
 import org.springframework.context.annotation.Profile
 import org.springframework.kafka.annotation.KafkaListener
 import org.springframework.kafka.support.Acknowledgment
 import org.springframework.stereotype.Service
-import java.time.LocalDateTime
 
 @Service
-@Profile("!integrasjonstest & !dev-postgres-preprod & !postgres")
```

Comment:
Er ikke dette den gamle listneren? Den må vi beholde

Reply:
Eller forvirret. Dette er kanskje den gamle. Hvorfor gjør du endringer på den?

Reply:
Nå skjønner jeg. Du har endret koden i den gamle til å være ny, og så har du lagd en ny klasse for den gamle. Som gjør det vanskelig å lese PR

Reply:
Stemmer på en måte. Valgte å rename den gamle og da tolker git det som en ny fil.

Reply:
git history:
![image](https://github.com/user-attachments/assets/977c8541-70c3-4e25-b521-ff869cf35dea)